src/Components/Album.jsx

```diff
@@ -1,11 +1,37 @@
-//import React from 'react';
-//import { AlbumName } from "./AlbumName";
+import React from "react";
+import { Albums } from "../App";
 //import { ArtistName } from "./ArtistName";
 //import { CoverImage } from "./CoverImage";
 
+//Extract album information
+export const extractedAlbums = Albums.map((album) => {
 
-export default function Album() {
+    const ArtistName = album.artists[0].name;
+    const albumName = album.name;
+    const coverImage = album.images[0].url;
```

Comment:
Remember that in React we start with the capitol letter feks AlbumName

I think all of these functions belong in the children components - children components only need to be imported to their parent (Album) component

Reply:
I think this would be the best component to create some HTML - but I also had to look at how others did it to understand more